WebService/ComparisonEngineUI/ComparisonEngineUI/Views/AddDrinkPage.xaml

```diff
@@ -0,0 +1,25 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://xamarin.com/schemas/2014/forms"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             x:Class="ComparisonEngineUI.Views.AddDrinkPage">
+    <ContentPage.Content>
+        <StackLayout>
+            <Picker Title="Select Bar"
+        TitleColor="Red">
+                <Picker.ItemsSource>
+                    <x:Array Type="{x:Type x:String}">
+                        <x:String>Baboon</x:String>
+                        <x:String>Capuchin Monkey</x:String>
+                        <x:String>Blue Monkey</x:String>
+                        <x:String>Squirrel Monkey</x:String>
+                        <x:String>Golden Lion Tamarin</x:String>
+                        <x:String>Howler Monkey</x:String>
+                        <x:String>Japanese Macaque</x:String>
```

Comment:
when did bars turn into monkeys :D

Reply:
an example list to see how it would look like on phone